apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -1776,6 +1776,13 @@ fn handle_error_internal<T: Into<opentelemetry::global::Error>>(
     #[cfg(test)]
     let threshold = Duration::from_millis(100);
 
+    // For now we have to suppress Metrics error: reader is shut down or not registered
```

Comment:
The reason that this had to move was that we do need to be able to see useful metrics errors, and this was suppressing them.